### DISCUSSION TITLE
Unlock the Romanised Artist & Romanised Title field in the editor by default

### DISCUSSION THREAD

**Discussion by Ryuusei-Aika** [ppy/osu#13484] [Ideas]
Currently, if the Artist and Title field contain only Unicode characters, the Romanised Artist and Romanised Title field would be forced to be the same with (resp.) Artist and Title, without the possibility of being changed (a.k.a. they are "locked"). This could lead to issues if the Artist has a preferred stylization that contains only Unicode characters.

For example:
A band named Left at London has a preferred stylization "/@/" as mentioned in [their official website](https://leftatlondon.com/). The slashes are representing both L words (Left and London). But right now, it is impossible to respect such stylization by putting /@/ in the Artist field and Left at London in the Romanised Artist field, since "/@/" doesn't contain non-Unicode characters, thus would not unlock the Romanized Artist field. On the other hand, it could cause issues for users with the "Prefer original metadata" setting enabled if we just put "Left at London" in both Artist fields.

Therefore, it would be appreciated that if the Romanised Artist and Romanised Title field could be unlocked by default in the editor (at least in stable). The auto-sync function between 2 Artist fields and 2 Title fields should still work normally, and this can help to resolve complicated metadata problems as mentioned above. This should also be a relatively simple fix code-wise.

**Comment by peppy** [comment]
That's not the intended use of romanization. It should be used when a non english language is used that generally cannot he read by the average user.
